test/f90_correct/bih-head.f90

```diff
@@ -0,0 +1,94 @@
+!
```

Comment:
Can we move this test to a new directory (test/ili_correct) since this tests the ILI output? I am OK if it is done in a subsequent patch.

Reply:
+1. I was going to write the exact same comment.